src/components/Calendar.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect} from 'react';
+import Weather from './Weather';
+import CalendarMonth from './CalendarMonth';
+import CalendarDay from './CalendarDay';
+import AddEvent from './AddEvent';
+
+
+export default function Calendar() {
+
+    const currentDay = new Date().getDate();
+    const currentMonth = new Date().getMonth();
+    const currentYear = new Date().getFullYear().toString();
+    const initialYearArr = [{}, {}, {},  {}, {}, {}, {}, {}, {}, {}, {}, {}];
```

Comment:
1. Here is another way to initialize and array with some default values of desired length
```suggestion
    const initialYearArr = Array(12).fill().map(() => ({});
```
2. Since this constant is used as initial value of `eventsMonthly` I guess it could be defined within `eventsMonthly` declaration statement, i.e., on line 23